lnrpc/routerrpc/router.proto

```diff
@@ -228,12 +258,18 @@ message ChannelUpdate {
 service Router {
     /**
     SendPayment attempts to route a payment described by the passed
-    PaymentRequest to the final destination. If we are unable to route the
-    payment, or cannot find a route that satisfies the constraints in the
-    PaymentRequest, then an error will be returned. Otherwise, the payment
-    pre-image, along with the final route will be returned.
+    PaymentRequest to the final destination. The call returns immediately.
```

Comment:
maybe specify that it returns after the payment has been successfully registered with the router,

Reply:
added